mkosi.md

```diff
@@ -287,30 +287,34 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
 
 `Distribution=`
 
-: Matches against the configured distribution. Multiple distributions may
-  be specified, separated by spaces. If multiple distributions are specified,
-  the condition is satisfied if the current distribution equals any of the
-  specified distributions.
+: Matches against the configured distribution. Multiple distributions may be
+  specified, separated by spaces. A single distribution match can be negated by
+  preceding the value with `!`. If multiple distributions are specified, the
+  condition is satisfied if the current distribution equals any of the specified
+  distributions.
 
 `Release=`
 
-: Matches against the configured distribution release. If this condition
-  is used and no distribution has been explicitly configured yet, the
-  host distribution and release are used. Multiple releases may be specified,
-  separated by spaces. If multiple releases are specified, the condition is
-  satisfied if the current release equals any of the specified releases.
+: Matches against the configured distribution release. If this condition is used
+  and no distribution has been explicitly configured yet, the host distribution
+  and release are used. A single release match can be negated by preceding the
+  value with `!`. Multiple releases may be specified, separated by spaces. If
+  multiple releases are specified, the condition is satisfied if the current
+  release equals any of the specified releases.
```

Comment:
Does it make sense to support negation here since we already have the comparison operators? Technically you can already negate here by just using the reverse comparison operator right?

Reply:
The rich comparisons are for image versions. For releases we can't really do version compare, since for Debian these will be codenames.

Reply:
Ah lol my bad misread this, but then the same argument I made for distributions applies here